website/content/docs/audit/index.mdx

```diff
@@ -40,6 +40,18 @@ Note that currently only strings coming from JSON or being returned in JSON are
 HMAC'd. Other data types, like integers, booleans, and so on, are passed
 through in plaintext.
 
+### Optional audit log flags
```

Comment:
These options do not apply to audit devices, but instead to secrets engines and auth methods, and are documented in the relevant commands.  It makes sense to alert the reader about the existence of these options, but they don't make sense in isolation, without the commands to which they apply.  How about linking to the auth_tune, auth_enable, secrets_tune, and secrets_enable pages where we document the options?

Reply:
Thanks for the feedback, Nick. After discussing with Andrew and taking your feedback into account, the following changes have been made. Please re-review, and let me know if you have additional feedback.